src/components/ble/NavigationService.h

```diff
@@ -27,11 +27,16 @@
 #undef min
 
 namespace Pinetime {
+
+  namespace System {
+    class SystemTask;
+  }
+
   namespace Controllers {
 
     class NavigationService {
     public:
-      NavigationService();
+      NavigationService(Pinetime::System::SystemTask& systemTask);
```

Comment:
This should be marked explicit so that it can't accidentally be used for casts.

Reply:
Yes, good point! Fixed.